tests/CreatesApplication.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests;
+
+use Illuminate\Contracts\Console\Kernel;
+use Illuminate\Foundation\Application;
+
+trait CreatesApplication
+{
+    public function createApplication(): Application
+    {
+        $app = require __DIR__ . "/../bootstrap/app.php";
```

Comment:
You don't need bootstrapping file. You can bootstrap application from `Blumilk\Meetup\Core\MeetupApplication` class.

Reply:
Then I get the RuntimeException "A facade root has not been set".

Reply:
What facade do you want to use and where?

Reply:
Nowhere, I just create object of MeetupApplication class and use getApplication method.

Reply:
I still don't get it. Behat tests were working fine with this packaged app approach. Please take care about removing `/bootstrap/app.php` file and wire application in your test cases bootstrap.